packages/dd-trace/test/appsec/index.spec.js

```diff
@@ -203,5 +204,92 @@ describe('AppSec Index', () => {
       }, context)
       expect(Reporter.finishAttacks).to.have.been.calledOnceWithExactly(req, context)
     })
+
+    it('should propagate incoming http end data with weird framework', () => {
+      const context = {}
+
+      sinon.stub(Gateway, 'getContext').returns(context)
+
+      const req = {
+        body: null,
+        query: 'string',
+        route: {},
+        params: 'string',
+        cookies: 'string'
+      }
+      const res = {
+        getHeaders: () => ({
+          'content-type': 'application/json',
+          'content-lenght': 42
+        }),
+        statusCode: 201
+      }
+
+      sinon.stub(Gateway, 'propagate')
+      sinon.stub(Reporter, 'finishAttacks')
+
+      AppSec.incomingHttpEndTranslator({ req, res })
+
+      expect(Gateway.getContext).to.have.been.calledOnce
+      expect(Gateway.propagate).to.have.been.calledOnceWithExactly({
+        'server.response.status': 201,
+        'server.response.headers.no_cookies': {
+          'content-type': 'application/json',
+          'content-lenght': 42
+        }
+      }, context)
+      expect(Reporter.finishAttacks).to.have.been.calledOnceWithExactly(req, context)
+    })
+
+    it('should propagate incoming http end data with express', () => {
+      const context = {}
+
+      sinon.stub(Gateway, 'getContext').returns(context)
+
+      const req = {
+        body: {
+          a: '1'
+        },
+        query: {
+          b: '2'
+        },
+        route: {
+          path: '/path/:c'
+        },
+        params: {
+          c: '3'
+        },
+        cookies: {
+          d: '4'
+        }
+      }
+      const res = {
+        getHeaders: () => ({
+          'content-type': 'application/json',
+          'content-lenght': 42
+        }),
+        statusCode: 201
+      }
+
+      sinon.stub(Gateway, 'propagate')
+      sinon.stub(Reporter, 'finishAttacks')
+
+      AppSec.incomingHttpEndTranslator({ req, res })
+
+      expect(Gateway.getContext).to.have.been.calledOnce
+      expect(Gateway.propagate).to.have.been.calledOnceWithExactly({
+        'server.response.status': 201,
+        'server.response.headers.no_cookies': {
+          'content-type': 'application/json',
+          'content-lenght': 42
+        },
+        // 'server.request.body': { a: '1' },
```

Comment:
Why are both the test and code for this commented out?

Reply:
Check out Vlad's first comment:
> Let's not do the body in this iteration to ensure good sleep for all of us (the binding's object limitation do not cover the size of objects, only their depth)

Reply:
Why keep the code commented out?

Reply:
I usually just do that when I already coded something and it's not needed immediately, and it's faster to just comment something than saving it in a branch. Do you really want me to remove it ?

Reply:
It's a good habit to avoid commented out code. If something is needed for later, moving to a separate branch that adds that functionality is a better approach.

Reply:
Done